docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -52,20 +75,28 @@ The WordPress [wp-env package](https://www.npmjs.com/package/@wordpress/env) let
 
 The `wp-env` package requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that the prerequisites are installed, install `wp-env` globally from the command-line using:
+After you have installed Docker, go ahead and install `wp-env` globally from the command-line using:
 
 ```sh
 npm -g install @wordpress/env
 ```
 
-This will be used in the next [Plugin step](wp-plugin.ms). To confirm it is installed and available, run:
+To confirm it is installed and available, run:
 
 ```sh
 wp-env --version
-> 1.4.0
+> 1.6.0
 ```
 
-### Alternatives
+The `wp-env` script is used to create a Docker WordPress environment, to start use:
+
+```sh
+wp-env start
+```
+
```

Comment:
This is what I see in terminal:
paaljoachimromdahl@Paals-MacBook-Pro ~ % wp-env --version
1.6.0
```
paaljoachimromdahl@Paals-MacBook-Pro ~ % wp-env start
✖ No .wp-env.json file found at '/Users/paaljoachimromdahl/.wp-env.json' and could not determine if '/Users/paaljoachimromdahl' is a WordPress installation, a plugin, or a theme.
paaljoachimromdahl@Paals-MacBook-Pro ~ % 
```

I noticed I have installed Docker but did not start it up. I started up Docker. It asked for privileged access.

<img width="622" alt="Screen Shot 2020-07-08 at 20 52 46" src="https://user-images.githubusercontent.com/5323259/86958823-0a785900-c15d-11ea-9e9a-3f16acb51eb4.png">

Clicking Ok.

<img width="464" alt="Screen Shot 2020-07-08 at 20 53 58" src="https://user-images.githubusercontent.com/5323259/86958916-372c7080-c15d-11ea-9df3-8ee5e078ac23.png">

Docker now starts up. Saying:
"Get started with Docker in a few easy steps."

There is a tutorial there which I clicked next to get into. I am not sure how it is related to wp-env.

<img width="1061" alt="Screen Shot 2020-07-08 at 20 56 11" src="https://user-images.githubusercontent.com/5323259/86959103-85417400-c15d-11ea-9bff-b5d5bb3c3381.png">

I went through the tutorial and end up with this screen.
<img width="1055" alt="Docker-after-tut-screen" src="https://user-images.githubusercontent.com/5323259/86959276-d487a480-c15d-11ea-88cc-37e8624b45b5.png">

Reply:
The fix for that is related to my earlier comment; basically wp-env needs to be run with either a local config file or from the directory of plugin/theme/core

Reply:
See the error message, it is trying to start in a plugin directory (or find .wp-env.json)
So you need to run it from inside the directory generated using:

`npx @wordpress/create-block gutenpride`

You would cd into `gutenpride` directory and then run `wp-env start`

The additional comments I just added from @noahtallen might help clarify.